portfolio/src/main/webapp/personalblog.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+        <!DOCTYPE html>
```

Comment:
nit: remove the leading space

Reply:
I was following the same formatting that I saw in the week 3 project folder, but span does make more sense.